networktest.go

```diff
@@ -252,16 +254,16 @@ func (l *lightningNode) stop() error {
 	// - start of the node wasn't initiated
 	// - process wasn't spawned
 	// - process already finished
-	processFinished := l.cmd.ProcessState != nil &&
-		l.cmd.ProcessState.Exited()
-	if l.cmd == nil || l.cmd.Process == nil || processFinished {
-		return nil
-	}
 
-	if runtime.GOOS == "windows" {
-		return l.cmd.Process.Signal(os.Kill)
+	select {
```

Comment:
The race was caused because of the accessing the `l.cmd.ProcessState`, for that reason I removed the logic which access this variable.